sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusAmqpConnection.java

```diff
@@ -44,5 +44,5 @@
      * @return A new or existing receive link that is connected to the given {@code entityPath}.
      */
     Mono<AmqpReceiveLink> createReceiveLink(String linkName, String entityPath, ReceiveMode receiveMode,
-        boolean isSession, String transferEntityPath, MessagingEntityType entityType);
+        boolean isSession, String sessionId, String transferEntityPath, MessagingEntityType entityType);
 }
```

Comment:
I know we had unused sessionenabled. But add another method rather than everything in this method